api/setup.py

```diff
@@ -32,11 +32,11 @@
         ]
     },
     install_requires=[
-        "coreapi",
-        "django~=3.0",
-        "djangorestframework",
-        "django-filter",
-        "drf-spectacular",
+        "coreapi~=2.3.3",
+        "django~=3.2.13",
+        "djangorestframework~=3.13.1",
+        "django-filter~=21.1.0",
+        "drf-spectacular~=0.22.1",
         "markdown",
```

Comment:
I was wondering why we need the markdown package here ?

Reply:
I can't remember. Think it is a dependency somewhere?

Reply:
It isn't used in our code at least. We could probably remove it and add it back if we need markdown.